src/articles/articles.service.ts

```diff
@@ -40,6 +41,7 @@ export class ArticlesService {
     private readonly commentsService: CommentsService,
     private readonly tagsService: TagsService,
     private readonly dbHelperRepository: DatabaseHelperRepository,
+    private readonly clapRepository: ClapRelationalRepository,
```

Comment:
Instead of injecting claps repository directly in article service please use it from claps service. Thanks

Reply:
sure! But now with this approach i don't actually need this so now it's been removed